py/server/deephaven/stream/kafka/consumer.py

```diff
@@ -275,19 +278,44 @@ def _consume(
     except Exception as e:
         raise DHError(e, "failed to consume a Kafka stream.") from e
 
+class ProtobufProtocol(JObjectWrapper):
+    """The protobuf serialization / deserialization protocol."""
+
+    j_object_type = jpy.get_type("io.deephaven.kafka.protobuf.Protocol")
+
+    @staticmethod
+    def serdes():
+        """The Kafka Protobuf serdes protocol. The payload's first byte is the serdes magic byte, the next 4-bytes are
+        the schema ID, the next variable-sized bytes are the message indexes, followed by the normal binary encoding of
+        the Protobuf data."""
+        return ProtobufProtocol(ProtobufProtocol.j_object_type.serdes())
+
+    @staticmethod
+    def raw():
```

Comment:
```suggestion
    def raw() -> ProtobufProtocol:
```

Reply:
Unfortunately, Python throws an error when trying to reference this class.

Reply:
Try `def raw() -> 'ProtobufProtocol':`